engines/python/setup/djl_python/deepspeed.py

```diff
@@ -236,17 +301,71 @@ def infer_task_from_model_architecture(self, config: PretrainedConfig):
                 f"Task could not be inferred from model config. "
                 f"Please manually set `task` in serving.properties.")
 
+    def get_model_pretrained(self, model_id_or_path, torch_dtype='auto', **kwargs):
+        tokenizer = AutoTokenizer.from_pretrained(model_id_or_path)
```

Comment:
please add some logic to allow `trust_remote_code` option

Reply:
also need to take care `revision`

Reply:
trust_remote_code and other parameter will get merged into **kwargs.
For example:
on line 384 and 391, 
```
            model, self.tokenizer, state_dict_mmap = self.load_model(
                self.model_id_or_path,
                'pretrained',
                self.ds_config['replace_with_kernel_inject'],
                low_cpu_mem_usage=self.low_cpu_mem_usage,
                trust_remote_code=self.trust_remote_code,
                **kwargs
            )
```
both 
```
low_cpu_mem_usage=self.low_cpu_mem_usage,
trust_remote_code=self.trust_remote_code,
```
will merge into **kwargs in self.load_model function
```
 def load_model(self, model_id_or_path, loading_method, use_mmap_loader, **kwargs):
```

Reply:
For AutoTokenizer, I didn't see **kwargs get passed in, following are the original code.
```
        if self.peft_config:
            self.tokenizer = AutoTokenizer.from_pretrained(
                self.peft_config.base_model_name_or_path)
        else:
            self.tokenizer = AutoTokenizer.from_pretrained(
                self.model_id_or_path)
```

Reply:
Did that not work before? What changed?

Reply:
Is that covered by `kwargs`?

Reply:
For example, model like https://huggingface.co/Salesforce/codegen25-7b-multi used a custom tokenizer rather than the built in one. So `revision` and `trust_remote_code` is necessary for the inputs

Reply:
Will address this use case in the followup PR. Probably need some refactor of the code